tracks/csharp/exercises/raindrops/README.md

```diff
@@ -0,0 +1,78 @@
+### Reasonable solutions
+#### LINQ
+
+```csharp
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+public static class Raindrops
+{
+    public static string Convert(int number)
+    {
+        Dictionary<int, string> dict = new Dictionary<int, string>()
+        {
+            [3] = "Pling",
+            [5] = "Plang",
+            [7] = "Plong",
+        };
+
+        var factors = dict
+            .Where(kv => number % kv.Key == 0)
+            .Select(kv => kv.Value)
+            .DefaultIfEmpty(number.ToString());
+
+        return String.Join(string.Empty, factors);
```

Comment:
I believe the C# coding conventions suggest to use `string.Join` (lowercase string).